packages/components/src/internal/app/utils.ts

```diff
@@ -354,17 +375,30 @@ export function isPlatesEnabled(moduleContext?: ModuleContext): boolean {
 
 export function isChartBuilderEnabled(moduleContext?: ModuleContext): boolean {
     return (
-        isLIMSEnabled(moduleContext) &&
-        resolveModuleContext(moduleContext)?.biologics?.[EXPERIMENTAL_CHART_BUILDER] === true
+        isFeatureEnabled(ProductFeature.ChartBuilding, moduleContext) &&
+        resolveModuleContext(moduleContext)?.samplemanagement?.[EXPERIMENTAL_CHART_BUILDER] === true
     );
 }
 
+// Should be enabled for LKS Starter, which won't have any product features
+export function isTransformScriptsEnabled(moduleContext?: ModuleContext): boolean {
+    return isFeatureEnabled(ProductFeature.TransformScripts, moduleContext) || hasPremiumModule(moduleContext);
+}
+
 export function isRReportsEnabled(moduleContext?: ModuleContext): boolean {
     return biologicsIsPrimaryApp(moduleContext);
 }
 
+export function isLKSSupportEnabled(moduleContext?: ModuleContext): boolean {
+    return isBiologicsEnabled(moduleContext) || hasPremiumModule(moduleContext);
+}
+
 export function isLIMSEnabled(moduleContext?: ModuleContext): boolean {
-    return biologicsIsPrimaryApp(moduleContext);
+    // Can't check for a module context module since this is defined in sampleManagement, so we base it on feature availability
+    // Currently neither chart builder nor transform scripts is a great choice because you need the experimental flag for chart builder
+    // and transform scripts may be enabled by virtue of having the premium module. We choose chart builder since it will give
+    // false negatives less often and can be updated via a setting.
+    return isSampleManagerEnabled(moduleContext) && isChartBuilderEnabled(moduleContext);
```

Comment:
thoughts on just having this directly check for the `isFeatureEnabled(ProductFeature.ChartBuilding, moduleContext)` instead of `isChartBuilderEnabled(moduleContext)`? The feature flag should be there regardless of if the experimental feature is enabled or not (though hopefully that exp flag goes away in 24.9 and this doesn't matter)

Reply:
Yes, that seems better. Thanks.